File: 4Week/16Day/과제/three.py
def solution(rsp):
    answer = ''
    list_rsp = list(rsp)
    for i in list_rsp:
        if i == '2':
            answer += '0'
        elif i == '0':
            answer += '5'
        elif i == '5':
            answer += '2'
    return answer

File: 4Week/17Day/java/Algo08/src/test/three.java
package test;

public class three {
	public String[] solution(String[] picture, int k) {
        String[] answer = new String[picture.length * k];
        int count = 0;
        for(int i = 0; i < picture.length; i++) {
        	String[] str = picture[i].split("");
        	StringBuilder sb = new StringBuilder();
        	
        	for(String s : str) {
        		for(int j = 0; j < k; j++) {
        			sb.append(s);
        		}
        	}
        	for(int j = 0; j < k; j++){
                answer[count++] = sb.toString();
            }
        }
        return answer;
    }
}
